src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -178,8 +178,10 @@ public async Task StartAsync(CancellationToken cancellationToken)
             {
                 await StartHostAsync(tokenSource.Token);
             }
-            catch (OperationCanceledException)
+            catch (OperationCanceledException ex)
             {
+                _logger.LogWarning("Host startup operation has been cancelled", ex);
```

Comment:
The comments at 191 indicates that getting into this catch block may not be an issue in certain cases. I wonder this warning messages causes confusion.

Reply:
looks like @fabiocav is the author of that comment :)

Reply:
Yup - Fabio and I recently discussed this and he disagrees with that comment xD This is why part 2 of this issue is to have a design discussion if we should be handling this differently. For now we want more data on what is causing the cancellation